### DISCUSSION TITLE
GitHub Universe is almost here 👾  [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#38224] [New to GitHub]
Less than a week away until we launch into orbit for GitHub Universe. Get ready and get excited with us! 

🌟**Community Superstars**: A round of applause for these community citizens 👏
* @Godnondsilva: Helped a new community member Pages 📄 user solve their [404 error](https://github.com/orgs/community/discussions/37215) with clear instructions and possible fixes - nice work! 
* @DenverCoder1: Joined the [conversation](https://github.com/orgs/community/discussions/37231) on personal access tokens with a simple answer and screenshot. 
* @MrCornelius: For [reporting](https://github.com/orgs/community/discussions/36818) a bug 🐛 in collaborator permissions with screenshots and excellent details. Way to go! 

🚀 **Rousing Reads and Fun Finds**: Check out cool contributions, improve your toolkit, and build games! 
* **Release Radar:** Hacktoberfest, hackathons, and open source contributions. @mishmanners [compiled](https://github.blog/2022-10-28-release-radar-sept-2022/) our favorite open source project ships. 
* **3 strategies for consolidating your toolkit and boosting productivity**: @chrisreddington and @mjpaulsen [explore](https://github.blog/2022-10-26-3-strategies-for-consolidating-your-toolkit-and-boosting-productivity/) how GitHub Enterprise can help you transform your software engineering organization and practices.
* **10th Annual GitHub Game Off Has Kicked Off**: Game Off is a "game jam" or hackathon encouraging newbie, amateur, indie, and AAA game developers to build games from scratch based on a theme 🎮. Each year we see thousands of people participate, hundreds of games getting created, and bajillions of new game source code hosted on GitHub. 

   It's a great excuse to try building your first game, learn a new language, and/or experiment with new shiny open source game engines. Unlike most game jams that are limited to 48-72 hours, Game Off lasts an entire month so [get involved](https://itch.io/jam/game-off-2022) 

🌜 **GitHub Universe is Next Week**: Join us in-person in San Francisco or virtually from anywhere 🌏
* **Choose your own GitHub Universe 2022 adventure**: Whether you’re just getting started or you’re a seasoned industry professional, there’s a session for you, [build your agenda](https://github.blog/2022-10-27-choose-your-own-github-universe-2022-adventure/) 🗓️. 
* **Be Prepared:** We’ve compiled a list of all your [frequently asked questions](https://githubuniverse.com/faq) ❓so you can be prepared! 
* **Register for GitHub Universe:** We hope to see you there, [sign-up](https://register.githubuniverse.com/) now. 

Stay tuned for special community content coming here during Universe next week! We hope to see you there 👀.

**Comment by SalamiAdebowale** [comment]
Please help me out

**Comment by YerimahYD** [comment]
As a new comer to Github can be challenging and frankly confusing. However, a few weeks from now I will look back at this post and probably laugh.

**Reply by jfullstackdev** [reply]
you might be confused with Git and GitHub, I too when I first encountered this, I thought they were the same thing or worst, why use them if you can code without them ? 

Git is the version control, think of any project we are doing, maybe just like a PowerPoint presentation, every time we check, there might be an error so we change it and sometimes, when we are in a hurry, we will be naming it randomly and repeatedly, `final`, `real final`, `at last final`, `final final final`, then you realized at some point you need one information that was deleted from the previous version and you have no way of going back to it to get that info

put that scenario in coding where every addition / deletion is critical, even a deleted `;` can ruin the entire project, so we need a critical version control, so we can efficiently keep track of all those, that is Git

plus, there should be a single source of truth when, say, in a tech company, all of you are coding in the same codebase or else, you and your team will be ending up debating all the time which version of codebase is correct, who overwrites the code, who did this buggy code etc., 

then, back then, collaboration would be mostly by sending patches through email, so that changes from other developers could be merged if maintainers would approve it but that was not so user-friendly and there was no central space, each project must have an online space where collaboration could take place and out of that necessity, GitHub was developed

now, we have all the added features of GitHub that can even be used in non-code setting, like this very GitHub Discussion feature, GitHub Pages, Project boards etc.,

**Reply by Favouritegithub** [reply]
Honestly, am so so confused with everything around here, and I just hope and pray all becomes different and normal in a few days time

**Comment by akasyaotomotiv** [comment]
thank you i need some help with this


11 Kas 2022 Cum 19:41 tarihinde Mir Habeel Ahmad ***@***.***>
şunu yazdı:
